<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendMsg</class>
 <widget class="QWidget" name="FriendMsg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Friendmsg{
    background:#D6AA9F;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>验证消息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>413</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
   color:white;
}
QLabel:hover{
   background-color:rgb(58,149,222);
   color:white;

}</string>
   </property>
   <property name="text">
    <string>一</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>443</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
   color:white;
}
QLabel:hover{
  background-color:rgb(212, 64, 39);
  color:rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>40</y>
     <width>481</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#F8F0EE;</string>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_2{
  border:1px solid #D1CAC8;
  border-radius:2px;
}</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_4{
   border-image:url(&quot;:/10.jpeg&quot;);
   border-radius:2px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_agree">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>35</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:2px;
border:1px solid #929291;
background:#FBFBFA;</string>
     </property>
     <property name="text">
      <string>同意</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>35</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:2px;
border:1px solid #929291;
background:#FBFBFA;</string>
     </property>
     <property name="text">
      <string>拒绝</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>25</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>姓名：来自星星的我</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>id：2014190130</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>65</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777207</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
background-color:rgb(248, 248, 248);
border:1px solid #929291;
border-radius:2px;
height:10px;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
 
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}
QComboBox::down-arrow {
	image: url(:/sys/arrow_d.png);
}
</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>添加到</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(&quot;:/friendmsg.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="rs/IM.qrc"/>
 </resources>
 <connections/>
</ui>
